widget/storybook/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@rango-dev/storybook",
+  "version": "0.31.0",
+  "license": "MIT",
+  "type": "module",
+  "main": "./storybook-static/index.html",
+  "private": true,
+  "files": [
+    "storybook-static",
+    "src"
+  ],
+  "scripts": {
+    "dev": "storybook dev -p 3000",
+    "build": "storybook build && cp ./vercel.json ./storybook-static",
+    "ts-check": "tsc --declaration --emitDeclarationOnly -p ./tsconfig.json",
+    "type-checking": "tsc --declaration --emitDeclarationOnly",
+    "clean": "rimraf dist",
```

Comment:
this should be `storybook-static` in your case

Reply:
This folder is created with yarn build

Reply:
you are removing `dist` which doesn't exist here. `yarn build` is creating a folder called `storybook-static`. so here shoudl be `rimraf storybook-static`